### PR TITLE
Fixes to ga code

### PR DESCRIPTION
**Pull request by AWare**
- Fix a typo
- Make ophan stuff async
- Add some guards 

@ajosephides

### CODE REVIEW

**Comment by AWare** [comment]
@paulbrown1982  @cb372

**Comment by ajosephides** [comment]
Fine with this to go with the other bug fixes but with the Ophan async fix this will double the number of server calls to Google Analytics.
This is something we must avoid as it will have a cost implication and so it may be be better if we can have this loaded synchronously.

File: assets/javascripts/src/modules/analytics/setup.js [hunk 1]
```diff
@@ -31,6 +31,9 @@ define([
     function setupAnalytics() {
         ophan.init();
         ga.init();
+        ophan.ophan.then(function(ophan){
+            ga.setOphanId(ophan.pageViewId);
```
Comment (paulbrown1982) [inline assets/javascripts/src/modules/analytics/setup.js]: Can't you just fire ga.init() here? If you only call 'setOphanId' here then it gets set after the pageview and won't be sent to Google. You'd have to fire pageview twice to make it send, which isn't appropriate.

**Comment by AWare** [comment]
@paulbrown1982

File: assets/javascripts/src/modules/analytics/setup.js [hunk 2]
```diff
@@ -31,6 +31,7 @@ define([
     function setupAnalytics() {
         ophan.init();
         ga.init();
+        ophan.loaded.then(ga.init,ga.init);
```
Comment (paulbrown1982) [inline assets/javascripts/src/modules/analytics/setup.js]: I'm just testing this, fwict it's firing both. Also you need to remove the ga.init() above!

File: assets/javascripts/src/modules/analytics/setup.js [hunk 3]
```diff
@@ -31,6 +31,7 @@ define([
     function setupAnalytics() {
         ophan.init();
         ga.init();
```
Comment (paulbrown1982) [inline assets/javascripts/src/modules/analytics/setup.js]: You need to remove this one!

**Comment by paulbrown1982** [comment]
👍

**Comment by AWare** [comment]
I think my renaming ophan.ophan to ophan.loaded has broken things. Will have another look in the morning.

**Comment by AWare** [comment]
That's better.
@paulbrown1982 ?

**Comment by paulbrown1982** [comment]
👍

**Pull request merged**